test/knative/values.yaml

```diff
@@ -54,5 +54,5 @@ global:
 
 
   proxy_init:
-    image: proxy_init
+    image: proxyv2
 
```

Comment:
This should be done in master as well

Reply:
fixed in https://github.com/istio/installer/pull/438